ext/lib/gui/lvgl/lv_conf.h

```diff
@@ -0,0 +1,464 @@
+
+#define USE_LV_THEME_TEMPL      0                           /*Just for test*/
+#define USE_LV_THEME_DEFAULT    CONFIG_LVGL_THEME_DEFAULT   /*Built mainly from the built-in styles. Consumes very few RAM*/
+#define USE_LV_THEME_ALIEN      CONFIG_LVGL_THEME_ALIEN     /*Dark futuristic theme*/
+#define USE_LV_THEME_NIGHT      CONFIG_LVGL_THEME_NIGHT     /*Dark elegant theme*/
+#define USE_LV_THEME_MONO       CONFIG_LVGL_THEME_MONO      /*Mono color theme for monochrome displays*/
+#define USE_LV_THEME_MATERIAL   CONFIG_LVGL_THEME_MATERIAL  /*Flat theme with bold colors and light shadows*/
+#define USE_LV_THEME_ZEN        CONFIG_LVGL_THEME_ZEN       /*Peaceful, mainly light theme */
+#define USE_LV_THEME_NEMO       CONFIG_LVGL_THEME_NEMO      /*Water-like theme based on the movie "Finding Nemo"*/
+
+#else
+
+#define LV_THEME_LIVE_UPDATE    0       /*1: Allow theme switching at run time. Uses 8..10 kB of RAM*/
+
+#define USE_LV_THEME_TEMPL      0       /*Just for test*/
+#define USE_LV_THEME_DEFAULT    0       /*Built mainly from the built-in styles. Consumes very few RAM*/
+#define USE_LV_THEME_ALIEN      0       /*Dark futuristic theme*/
+#define USE_LV_THEME_NIGHT      0       /*Dark elegant theme*/
+#define USE_LV_THEME_MONO       0       /*Mono color theme for monochrome displays*/
+#define USE_LV_THEME_MATERIAL   0       /*Flat theme with bold colors and light shadows*/
+#define USE_LV_THEME_ZEN        0       /*Peaceful, mainly light theme */
+#define USE_LV_THEME_NEMO       0       /*Water-like theme based on the movie "Finding Nemo"*/
+
+#endif
+
+/*==================
+ *    FONT USAGE
+ *===================*/
+
+/* More info about fonts: https://littlevgl.com/basics#fonts
+ * To enable a built-in font use 1,2,4 or 8 values
+ * which will determine the bit-per-pixel */
+#ifdef CONFIG_LVGL_DEFAULT_FONT_BUILD_IN_10_ASCII
+#define LV_FONT_DEFAULT &lv_font_dejavu_10
+#elif defined(CONFIG_LVGL_DEFAULT_FONT_BUILD_IN_20_ASCII)
+#define LV_FONT_DEFAULT &lv_font_dejavu_20
+#elif defined(CONFIG_LVGL_DEFAULT_FONT_BUILD_IN_30_ASCII)
+#define LV_FONT_DEFAULT &lv_font_dejavu_30
+#elif defined(CONFIG_LVGL_DEFAULT_FONT_BUILD_IN_40_ASCII)
+#define LV_FONT_DEFAULT &lv_font_dejavu_40
+#elif defined(CONFIG_LVGL_DEFAULT_FONT_BUILD_IN_MONOSPACE)
+#define LV_FONT_DEFAULT &lv_font_monospace_8
+#elif defined(CONFIG_LVGL_DEFAULT_FONT_CUSTOM)
+extern void *lv_default_font_custom_ptr;
+#define LV_FONT_DEFAULT ((lv_font_t*) lv_default_font_custom_ptr)
+#else
+#error No defulat font defined
+#endif
+
+#ifdef CONFIG_LVGL_BUILD_IN_FONT_10
+
+#define USE_LV_FONT_DEJAVU_10              CONFIG_LVGL_BUILD_IN_FONT_10_ASCII_BPP
+#define USE_LV_FONT_DEJAVU_10_LATIN_SUP    CONFIG_LVGL_BUILD_IN_FONT_10_LATIN_SUP_BPP
+#define USE_LV_FONT_DEJAVU_10_CYRILLIC     CONFIG_LVGL_BUILD_IN_FONT_10_CYRILLIC_BPP
+#define USE_LV_FONT_SYMBOL_10              CONFIG_LVGL_BUILD_IN_FONT_10_SYMBOL_BPP
```

Comment:
But `_SYMBOL_` here (note the missing "S"). The same is done for the other fonts.